docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -70,28 +70,42 @@ Finally, you can query the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``sqlserver`` in the above examples.
 
+.. _sqlserver-performance:
+
+Performance
+-----------
+
+The connector includes a number of performance improvements, detailed in the
+following sections.
+
+* Support for :ref:`JDBC connection pooling <base-jdbc-connection-pooling>`
+
 .. _sqlserver-pushdown:
 
 Pushdown
---------
-
-The connector supports :doc:`pushdown </optimizer/pushdown>` for processing the
-following aggregate functions:
-
-* :func:`avg`
-* :func:`count`
-* :func:`max`
-* :func:`min`
-* :func:`sum`
-* :func:`stddev`
-* :func:`stddev_pop`
-* :func:`stddev_samp`
-* :func:`variance`
-* :func:`var_pop`
-* :func:`var_samp`
+^^^^^^^^
+
+The connector supports pushdown for a number of operations:
+
+* :ref:`Join pushdown <base-jdbc-join-pushdown>`
+* :ref:`TopN pushdown <base-jdbc-topn-pushdown>`
+* :ref:`Aggregate pushdown <base-jdbc-aggregation-pushdown>` for the following
+  functions:
+
+    * :func:`avg`
+    * :func:`count`
+    * :func:`max`
+    * :func:`min`
+    * :func:`sum`
+    * :func:`stddev`
+    * :func:`stddev_pop`
+    * :func:`stddev_samp`
+    * :func:`variance`
+    * :func:`var_pop`
+    * :func:`var_samp`
 
 Data compression
-----------------
+^^^^^^^^^^^^^^^^
```

Comment:
Do `^^^`s mean it's a sub-item of "Performance"?
i think it shouldn't be.

Reply:
it does mean that.. if it should not be under performance.. does it belong higher up under Configuration?

Reply:
No, it's not (connector's) configuration.

Reply:
So we plan to have the following top level segments for each connector

Configuration
Performance
Security
SQL support
Limitation

From these I think Data compression either falls into configuration or performance. I dont think it deserves a separate top level entry. Any suggestion?